shared/chat/conversation/attachment-get-titles/index.js

```diff
@@ -109,7 +109,9 @@ class GetTitles extends React.Component<Props, State> {
           <Box style={isMobile ? stylesMobile : stylesDesktop}>
             <Box style={{...globalStyles.flexBoxCenter, height: 150, width: 150}}>
               {info.type === 'image' ? (
-                <Image
+                <OrientedImage
+                  preview={true}
+                  onLoad={() => {}}
```

Comment:
can we make onLoad be optional so we don't need to do this

Reply:
Got some strange Flow errors last time, I'll give it another try.